runner/src/server/plugins/logging.ts

```diff
@@ -1,4 +1,70 @@
-import pluginLogging from "../../../../digital-form-builder/runner/src/server/plugins/logging";
+import pino from "hapi-pino";
+import {config} from "./utils/AdapterConfigurationSchema";
+
+export default {
+    plugin: pino,
+    options: {
+        prettyPrint: false,
```

Comment:
If JSON printing is the default maybe it suffices to simply remove any reference to prettyPrint? See [here](https://github.com/pinojs/pino/blob/619e049aed18dd79a9fdfac5dceab86c34e9bda0/docs/pretty.md) where it mentions "_By default, Pino log lines are newline delimited JSON (NDJSON). This is perfect for production usage and long-term storage._"